src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1527,7 +1538,7 @@ private void onDecodeError(
             }
             else
             {
-                doEncodeConnack(traceId, authorization, reasonCode);
+                doEncodeConnack(traceId, authorization, reasonCode, EMPTY_OCTETS.buffer(), 0, 0);
```

Comment:
Either `EMPTY_BUFFER` constant directly instead, or consider changing last 3 parameters of `doEncodeConnack` to a single `Flyweight` parameter instead.